### PR TITLE
Pre commit changes i

### PR DESCRIPTION
**Pull request by phracek**
This commit review main.py script checked by pre-commit checker.

The rest will follow as soon as this PR will be merged.

### CODE REVIEW

File: container_workflow_tool/main.py [hunk 1]
```diff
@@ -211,18 +219,20 @@ def _get_images(self):
...
             return base
 
-    def _prebuild_check(self, image_set, branches=[]):
+    def _prebuild_check(self, image_set: List, branches: List = None):
         tmp = self._get_tmp_workdir(setup_dir=False)
         if not tmp:
-            raise RebuilderError("Temporary directory structure does not exist. Pull upstream first.")
+            raise RebuilderError(
+                "Temporary directory structure does not exist. Pull upstream first."
+            )
```
Comment (pkubatrh) [inline container_workflow_tool/main.py:235]: I would rather use the previous version here as I do not really like the hanging indent style used. If needed you can use a new variable (eg. `msg`) to assign the output into if line length is the issue.

Same for other instances further down the diff.

File: container_workflow_tool/main.py [hunk 2]
```diff
@@ -426,12 +433,10 @@ def list_images(self):
 
     def print_upstream(self):
         """Prints the upstream name and url for images used in config"""
-        template = "{component} {img_name} {ups_name} {url}"
         for i in self._get_images():
-            ups_name = re.search(".*\/([a-zA-Z0-9-]+).git",
+            ups_name = re.search(".*\\/([a-zA-Z0-9-]+).git",
```
Comment (pkubatrh) [inline container_workflow_tool/main.py:437]: Not sure I about the change here. I think the `\` was used as an escape for `/`. Though now I am not sure if escaping `/` is even necessary. If not I would just remove the `\` entirely.

File: container_workflow_tool/main.py [hunk 3]
```diff
@@ -545,11 +556,14 @@ def dist_git_changes(self, rebase=False):
         self.distgit.dist_git_changes(images, rebase)
         self.logger.info("\nGit location: " + tmp)
         if self.args:
-            template = "./rebuild-helper {} git show"
+            tmp_str = ' --tmp ' + self.tmp_workdir if self.tmp_workdir else '"'
             self.logger.info("You can view changes made by running:")
-            self.logger.info(template.format('--base ' + self.base_image + (' --tmp ' + self.tmp_workdir if self.tmp_workdir else "")))
+            self.logger.info(f"cwt|rh_cwt --base {self.base_image} {tmp_str} git show")
```
Comment (pkubatrh) [inline container_workflow_tool/main.py:561]: I would say using rhcwt here would just confuse anyone not knowing of its existence :)

**Comment by pkubatrh** [comment]
Thanks for the cleanup! Just some minor issues in separate comments

**Comment by phracek** [comment]
@pkubatrh I have addressed all issues.

File: container_workflow_tool/main.py [hunk 4]
```diff
@@ -529,7 +539,9 @@ def push_changes(self):
         self._check_kerb_ticket()
         tmp = self._get_tmp_workdir(setup_dir=False)
         if not tmp:
-            raise RebuilderError("Temporary directory structure does not exist. Pull upstream/rebase first.")
+            raise RebuilderError(
+                "Temporary directory structure does not exist. Pull upstream/rebase first."
+            )
```
Comment (pkubatrh) [inline container_workflow_tool/main.py:544]: Hanging indent still used here

**Comment by pkubatrh** [comment]
Otherwise lgtm, thanks!

**Pull request merged**